src/pip/_internal/vcs/subversion.py

```diff
@@ -213,14 +213,16 @@ def call_vcs_version(self):
         #      compiled Feb 25 2019, 14:20:39 on x86_64-apple-darwin17.0.0
         #   svn, version 1.7.14 (r1542130)
         #      compiled Mar 28 2018, 08:49:13 on x86_64-pc-linux-gnu
+        #   svn, version 1.12.0-SlikSvn (SlikSvn/1.12.0)
```

Comment:
Hi :-) Would it be worth adding this case to the existing tests?
https://github.com/pypa/pip/blob/930aa5c70855a150a98503119d18536f91dfcc61/tests/unit/test_vcs.py#L442-L454

Reply:
Thanks, I didn’t know the test exists. Added.